hack/local-setup.sh

```diff
@@ -205,7 +212,7 @@ deployAgentSecret() {
 
   kubectl create namespace mctc-system || true
 
-  makeSecretForCluster $KIND_CLUSTER_CONTROL_PLANE $(kubectl config current-context) $localAccess |
+  makeSecretForCluster $KIND_CLUSTER_CONTROL_PLANE $clusterName $localAccess |
```

Comment:
@philbrookes @sergioifg94 Could you check this please. Currently the secrets on the data and control plane differ in that they both have different cluster names in them. This removes the "kind-" prefix and makes it consistent in both scerets.